launcher/resources/backgrounds/backgrounds.qrc

```diff
@@ -1,8 +1,11 @@
 <!DOCTYPE RCC>
 <RCC version="1.0">
     <qresource prefix="/backgrounds">
-        <file alias="kitteh">catbgrnd2.png</file>
-        <file alias="catmas">catmas.png</file>
-        <file alias="cattiversary">cattiversary.png</file>
+        <file alias="kitteh">kitteh.png</file>
+        <file alias="kitteh-xmas">kitteh-xmas.png</file>
+        <file alias="kitteh-bday">kitteh-bday.png</file>
+        <file alias="rory">rory.png</file>
+        <file alias="rory-xmas">rory-xmas.png</file>
+        <file alias="rory-bday">rory-bday.png</file>
```

Comment:
Need to add flat rory

Reply:
forgot to add that to the commit, thanks!